tools/install_golangci_lint_deps.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
All the other tools use the explicit `#!/bin/bash` here it seems, unless there is a good reason I would keep it this way. I personally prefer the `#!/bin/sh`  (or `#!/bin/bash`) because it ensures it's always the distro provided bash [0] that is used, not some random one installed in $PATH. But then I'm coming from a distro background so take it with a grain of salt :)

[0] assuming no "make install prefix=/" was used

Reply:
I thought `/usr/bin/env` is the most generic way but the main reason was, that my IDE suggested and autocompleted that line 🙊

Reply:
I'd rather have a small chat in the team about the way to go and I'd be happy to align all then.

Reply:
you are right about the `sh` though - `bash` is not needed in this script ;)